src/nvim/version.c

```diff
@@ -67,6 +67,7 @@ NULL
 
 // clang-format off
 static const int included_patches[] = {
+  3018,
```

Comment:
You'll want to remove this.
Updates to `included_patches` is managed automatically (and only contains patches from v8.0.x currently).

Reply:
@seandewar Ah thanks, I've removed the line.